rpc/flipt/auth/auth.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+package flipt.auth;
+
+option go_package = "go.flipt.io/flipt/rpc/flipt/auth";
+
+option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
+  info: {
+    title: "Flipt Authentication APIs";
+    version: "latest";
+    contact: {
+      name: "Flipt Team";
+      url: "https://github.com/flipt-io/flipt";
+      email: "dev@flipt.io"
+    };
+    license: {
+      name: "MIT License";
+      url: "https://github.com/flipt-io/flipt/blob/main/rpc/flipt/LICENSE";
+    };
+  };
+  external_docs: {
+    url: "https://www.flipt.io/docs";
+    description: "Flipt Docs";
+  }
+  schemes: HTTP;
+  schemes: HTTPS;
+  consumes: "application/json";
+  produces: "application/json";
+};
+
+enum Method {
+  METHOD_TOKEN = 0;
```

Comment:
do we want to make the default 'unknown' or 'none'? https://docs.buf.build/lint/rules#enum_zero_value_suffix 

This is something I didn't do previously that I regret